src/main/java/application/App.java

```diff
@@ -1,21 +1,164 @@
 package application;
 
+import java.util.Random;
+import java.util.Scanner;
+
 public class App {
-	public static void main(String[] args) {
-
-	}
-
-	/*
-	 * for the bonus task
-	 *
-	 * private static int generateID() {}
-	 *
-	 */
-
-	/*
-	 * for the console application
-	 *
-	 * private static void application() {}
-	 *
-	 */
+    static int numberOfIds = 0;
+    static int numberOfDepartments = 0;
+    static int numberOfEmployees = 0;
+    static Random rand = new Random();
+
+
+
+    public static void main(String[] args) {
+        int[] ids = new int[1]; // generate array containing created IDs
+        Department[] departments = new Department[1]; // generate array containing departments
+        Employee[] employees = new Employee[1]; // generate array containing employees

```

Comment:
Make bigger arrays, they will be probably be bigger and therefore you save some copying.